_Sidebar.md

```diff
@@ -17,7 +17,7 @@
   * Getting started with the codebase
     * [[Populating data on local server|Populating-data-on-local-server]]
     * Tutorials
-      * [[Tutorial 1|Tutorial-1]]
+      * [[Make a Simple UI change|Tutorial-1]]
```

Comment:
Why capitalize "Simple" but not "change"?

Suggest just keeping both of those lowercase.

Reply:
Done.